bin/run_tests.py

```diff
@@ -3,28 +3,13 @@
 import os
 import subprocess
 import sys
-from glob import glob
 
 if __name__ == '__main__':
     # move cwd to the project root
     os.chdir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
     # run the unit tests
-
     subprocess.check_call([sys.executable, '-m', 'pytest', 'unit_test'])
 
     # run the integration tests
-
-    test_projects = sorted(glob('test/??_*'))
-
-    if len(test_projects) == 0:
-        print('No test projects found. Aborting.', file=sys.stderr)
-        exit(2)
-
-    print('Testing projects:', test_projects)
-
-    run_test_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'run_test.py')
-    for project_path in test_projects:
-        subprocess.check_call([sys.executable, run_test_path, project_path])
-
-    print('%d projects built successfully.' % len(test_projects))
+    subprocess.check_call([sys.executable, '-m', 'pytest', '--durations', '0', 'test'])
```

Comment:
Oh, another one: we might want to add `-x` or `--max-fail` here, to save on CI time? It's tricky as there can be advantages to getting a list of all failed builds at once, but I never heard anyone complain about the current situation (with `check_call` in a for-loop).

Reply:
Good idea - `test_0_basic` is named to run first and catch simpler errors - better to stop and give quick feedback I reckon.